docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -59,11 +59,14 @@ of all *other* nodes in your BigchainDB cluster
 * If you're deploying the first node in the cluster,
   the value should be ``""`` (an empty string).
 * If you're deploying the second node in the cluster,
-  the value should be one public key inside double quotes.
+  the value should be the BigchainDB public key of the first/original
+  node in the cluster, inside double quotes.
```

Comment:
Okay I will remove the "inside double quotes" prepositional phrase (both instances).